src/SME.SGP.Aplicacao/Consultas/ConsultasAulaPrevista.cs

```diff
@@ -58,7 +58,7 @@ public async Task<AulasPrevistasDadasAuditoriaDto> ObterAulaPrevistaDada(Modalid
             var usuarioLogado = await mediator.Send(new ObterUsuarioLogadoQuery());
             var codigoTerritorioCorrespondente = (long?)null;
 
-            var rf = usuarioLogado.EhPerfilDRE() || usuarioLogado.EhPerfilSME() ? string.Empty : usuarioLogado.CodigoRf;
+            var rf = usuarioLogado.EhPerfilDRE() || usuarioLogado.EhPerfilSME() || usuarioLogado.EhProfessor() ? string.Empty : usuarioLogado.CodigoRf;
```

Comment:
Em caso de território é necessário que seja considerado o professor para buscar o código equivalente. Sem isso corre o risco de professores diferentes que tenham o mesmo território compartilhem as mesmas informações indevidamente.

Reply:
Verdade, irei colocar essa condição de ignorar o RF só quando não for território, pois sem isso, acaba acontecendo o problema do bug.